stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -567,11 +567,12 @@ extension Array : _ObjectiveCBridgeable {
 
 extension NSDictionary : ExpressibleByDictionaryLiteral {
   public required convenience init(
-    dictionaryLiteral elements: (NSCopying, AnyObject)...
+    dictionaryLiteral elements: (_Hashable, Any)...
```

Comment:
A generic `<T: Hashable>` initializer wouldn't be faithful to NSDictionary, which allows heterogeneous keys. If we end up doing `AnyHashable` subtyping, maybe we could use that instead, though.